starter-code/scripts/controllers/articleController.js

```diff
@@ -35,6 +37,7 @@
   };
 
   // COMMENT: What does this method do?  What is it's execution path?
+  // This has the same functionality as the author filter, but handles when the category filter dropdown is selected. The selection is filtered by all articles by category and then the articleController is used to render the selected articles to the page.
```

Comment:
Same as above. One thing to keep in mind is that articleController.loadByAuthor and articleController.loadByCategory is **ITSELF** not finding the articles with a given value -- it's calling article.findWhere which does this work. The job of these methods is really calling article.findWhere, passing the appropriate values in, and then defining and providing article.findWhere with a callback function, articleData (that attaches the articles returned from the query onto the ctx object and then calls the next function in the middleware chain).